src/LinuxTracing/PerfEventReaders.cpp

```diff
@@ -18,12 +18,102 @@
 #include "OrbitBase/Logging.h"
 #include "OrbitBase/MakeUniqueForOverwrite.h"
 #include "OrbitBase/ThreadUtils.h"
+#include "PerfEvent.h"
 #include "PerfEventOrderedStream.h"
 #include "PerfEventRecords.h"
 #include "PerfEventRingBuffer.h"
 
 namespace orbit_linux_tracing {
 
+enum class ReadingFlags : int {
```

Comment:
Please use the `perf_event_sample_format` flags defined in `perf_event.h`. For usage see e.g. `PERF_SAMPLE_REGS_USER` in `PerfEventOpen.cpp`.

Reply:
Done